src/main/scala/io/github/ilyazinovich/dmmf/PriceOrder.scala

```diff
@@ -1,19 +1,22 @@
 package io.github.ilyazinovich.dmmf
 
 import cats.data.NonEmptyList
+import cats.syntax.semigroup._
+import cats.syntax.either._
+import cats.instances.either._
+import cats.instances.list._
 
 object PriceOrder {
 
   type GetProductPrice = ProductCode => Price
 
   def priceOrder(order: Order, getProductPrice: GetProductPrice): Either[NonEmptyList[Error], PricedOrder] = {
-    calculateOrderLinesPrice(order.orderLines, getProductPrice).right.flatMap { pricedOrderLines =>
-      val prices = pricedOrderLines.map(pricedOrderLine => pricedOrderLine.price)
-      BillingAmount.total(prices) match {
-        case Left(error) => Left(NonEmptyList.of(error))
-        case Right(billingAmount) =>
-          Right(PricedOrder(order.orderId, order.customerInformation, pricedOrderLines, billingAmount))
-      }
+    calculateOrderLinesPrice(order.orderLines, getProductPrice).flatMap { pricedOrderLines =>
+      val prices = pricedOrderLines.map(_.price)
+      BillingAmount.total(prices).bimap(
+        NonEmptyList.of(_),
+        { billingAmount => PricedOrder(order.orderId, order.customerInformation, pricedOrderLines, billingAmount) }
```

Comment:
Why do we need curly braces here? The code looks nice without them.

Reply:
We don't